journal/modernized/32-epistles.adoc

```diff
@@ -98,7 +99,11 @@ that we through him may be heirs of an everlasting inheritance,
 and not go after other lovers, or have the heart carried away with them,
 who will not stand instead, nor can they save in the needful time.
 
-William Edmundson.Jamaica, the 24th of the Twelfth month, 1671.
+[.signed-section-signature]
+William Edmundson
+
+[.signed-section-context-close]
+Jamaica, the 24th of the Twelfth month, 1671.
 
 === Part of a Letter to His Wife.
```

Comment:
Ok, so because this is introducing a letter, and telling you who it is addressed to, then I would also use [.letter-heading] here instead of a === subheading.   LIke this:

```ASCIIDOC
[.letter-heading]
Part of a Letter to His Wife
```